src/runtime/analytics-service.js

```diff
@@ -72,6 +75,15 @@ export class AnalyticsService {
 
     /** @private {?Promise} */
     this.lastAction_ = null;
+
+    /** @private @const {!../api/client-event-manager-api.ClientEventManagerApi} */
+    this.eventManager_ = deps.eventManager();
+    this.eventManager_.registerEventListener(
+        this.handleClientEvent_.bind(this));
+
+    /** @private @const {!boolean} */
+    this.logPropensityEvents_ = isExperimentOn(deps.win(),
```

Comment:
again, not enough protection. Maybe add a todo using the same bug
or introduce a method
enableLoggingForPropensity()

Ultimately we'll have to think of config names that the publisher can update. Will add a comment on the bug to work with Shams on closing the loop on this.

Reply:
Ok